index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link href="styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Source+Serif+Pro:wght@400;700;900&display=swap" rel="stylesheet">
+  <title>Yiwen's resume</title>
+
+</head>
+
+<body>
+
+  <h1>Yiwen Yu</h1>
+  <h2>yyu01@mica.edu | 443-854-7551</h2>
```

Comment:
While I think you can keep the styles on this as is, I would question if your contact info is heading-level info? And, even if so, they should be separated out into separate `h2` elements because, while they're related, they're not really one thing:

```suggestion
  <h2>yyu01@mica.edu</h2>
  <h2>443-854-7551</h2>
```